src/database/models.py

```diff
@@ -18,19 +19,24 @@ class User(Base):
     creation_timestamp = Column(DateTime(timezone=True), nullable=False)
     modify_timestamp = Column(DateTime(timezone=True), nullable=False)
     is_active = Column(Boolean, nullable=False)
-    
+    invitations = relationship("Invitation", back_populates="user")
```

Comment:
I'm sorry for the weird question but is it `user` or it should be `user_id`?

Reply:
`user` is the name of the field to populate with the relevant invitations.